tailwind.config.js

```diff
@@ -54,6 +54,7 @@ module.exports = {
 
       "search-keyboard-cursor": "#0C3B8C", // background for cursor highlighting in keyboard navigable areas (e.g. package search dropdown)
       "search-term-highlight": "rgb(221, 232, 251)",
+      "tab-overview": "rgba(211, 213, 249, 0.60)",
```

Comment:
This color seems to occur in the designs in more places than just the Learn Area tabs. Here's a potentially better interim name until we choose one in the Figma documents:

```suggestion
      "learn-area-light-blue": "rgba(211, 213, 249, 0.60)",
```